src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2561,6 +2562,7 @@ static bool deleteContextAttributeItem
   bool*                     entityModified,
   orion::BSONDate*          dateExpiration,
   ApiVersion                apiVersion,
+  const bool&               overrideMetadata,
```

Comment:
It seems this parameter is not used inside the function. Thus, it should be removed (also from L2750).

Reply:
> It seems this parameter is not used inside the function. Thus, it should be removed (also from L2750).

removed in https://github.com/telefonicaid/fiware-orion/pull/4186/commits/7a65459e06212144bc3932eca95122f14ce9f3f9